openedx_webhooks/config.py

```diff
@@ -3,13 +3,7 @@
 
 class DefaultConfig:
     SECRET_KEY = os.environ.get("FLASK_SECRET_KEY", "secrettoeveryone")
-    SQLALCHEMY_DATABASE_URI = os.environ.get("DATABASE_URL", "sqlite:///openedx_webhooks.db")
```

Comment:
I thought about this, and am not sure of the best approach.  DefaultConfig is accessed in a Flask-centric way, which felt odd in files that otherwise had no Flaskiness in them.

Reply:
sounds good, I think this is fine for now.